src/components/prescription-details.scss

```diff
@@ -5,7 +5,7 @@
 .prescriptionContainer {
   display: flex;
   flex-direction: column;
-  width: 100%;
+  max-width: 80%;
```

Comment:
I did this because I noticed that the medication event card width was different between the "Prescription Details" and "History and Comments" view without this.... the max width percentage is set to 80% in the prescription details SCSS file.